cluster/v1beta1/helpers.go

```diff
@@ -82,3 +86,23 @@ func BuildClusterSelector(clusterSet *ManagedClusterSet) (labels.Selector, error
 		return nil, fmt.Errorf("selectorType is not right: %s", clusterSet.Spec.ClusterSelector.SelectorType)
 	}
 }
+
+// GetDecisionsOfPlacement return the PlacementDecisions generated by the Placement
+func GetDecisionsOfPlacement(placement *Placement, decisionGetter PlacementDecisionGetter) ([]*PlacementDecision, error) {
```

Comment:
I am not sure if it is helpful. I think you would want to return a sets.String with all cluster names. Considering it should be used during schedule, you probably want to return the clusters added to the decision, the clusters removed from the decisions.

Reply:
@qiujian16 yes, update this func to GetDecisionClustersOfPlacement which will return cluster names of one placement's decision.